src/components/dev-hub/survey-banner.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import Button from './button';
+import Icon from '@leafygreen-ui/icon';
+import { fontSize, screenSize, size, colorMap } from './theme';
+
+const Footer = styled('div')`
+    display: ${({ showSurvey }) => (showSurvey ? `block` : `none`)};
+    background-color: ${colorMap.greyDarkTwo};
+    position: fixed;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    border: 1px solid ${colorMap.greyDarkOne};
+`;
+
+const FooterContent = styled('div')`
+    display: flex;
+    justify-content: space-between;
+    @media ${screenSize.mediumAndUp} {
+        margin: 0 2% 0 15%;
```

Comment:
We have two children 
1. The whole survey text and survey buttons
2. Close banner icon
one should be centered with the parent div and second one should float right.may be I need to get rid of space between and do float right.

Reply:
Is it possible for us to move away from 2% and 15%? These seem a bit hacky to me?